src/index.html

```diff
@@ -9,6 +9,18 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+
+    <form class="search-form" data-qa="big">
+      <label>
+        <input type="text" class="search_bar-input" name="search-big" placeholder="Try &quot;Los Angeles&quot;" data-qa="keypress">
+      </label>
+    </form>
+
+    <form class="search-form" data-qa="small">
+      <label>
```

Comment:
Its not cool that your label smaller then input, maybe apply to label display: block

<img width="687" alt="Screen Shot 2020-02-27 at 10 41 52 AM" src="https://user-images.githubusercontent.com/50083082/75426678-d2127400-594d-11ea-8755-013f8943ca29.png">